models.py

```diff
@@ -6,9 +6,45 @@
 
 from data_pipeline.glove_embeddings import get_weight_matrix
 
+# this size came from the highest code of a character that was found in the data set
+# の has the code 12398
+NUMBER_OF_ASCII_CHARACTERS = 12400
+
 #TODO: move this.
 BOS_IDX = 1
 
+class CharacterLevelEmbedding(nn.Module):
```

Comment:
Would it be a good idea to have this class in a separate file? The models file is pretty crowded as it is..